LICENSE

```diff
@@ -1,6 +1,8 @@
 MIT License
 
-Copyright (c) 2016-2020 Matteo Collina
+Copyright (c) 2016-2023 The Fastify Team
```

Comment:
```suggestion
Copyright (c) 2020-2023 The Fastify Team
Copyright (c) 2016-2020 Matteo Collina
```

Because this was migrated to the org in 2020, ref https://github.com/fastify/avvio/issues/109.